README.md

```diff
@@ -16,23 +16,8 @@ major standard library compilers.
 * [Give *std::optional* Range Support (P3168R2)](https://wg21.link/P3168R2)
 * [`std::optional<T&>` (P2988R5)](https://wg21.link/P2988R5)
 
-## Table of Contents
-
-* [beman.optional26: C++26 Extensions for std::optional](#bemanoptional26-c26-extensions-for-stdoptional)
-  * [Table of Contents](#table-of-contents)
-  * [License](#license)
```

Comment:
did you intend to remove the rest of this?

Reply:
Yes. At some point me and @camio  had a discussion that GitHub UI generates a TOC anyways.

Reply:
I see thanks!